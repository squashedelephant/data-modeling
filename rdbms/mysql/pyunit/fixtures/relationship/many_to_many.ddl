CREATE TABLE employee(id INT NOT NULL AUTO_INCREMENT, first_name CHAR(15) NOT NULL, last_name CHAR(15) NOT NULL, PRIMARY KEY (id));
CREATE TABLE dept(id INT NOT NULL AUTO_INCREMENT, name VARCHAR(12), acct_code INT UNSIGNED, PRIMARY KEY (id));
CREATE TABLE employee_dept(emp_id INT NOT NULL, dept_id INT NOT NULL, PRIMARY KEY (emp_id, dept_id), FOREIGN KEY (emp_id) REFERENCES employee(id), FOREIGN KEY (dept_id) REFERENCES dept(id));
