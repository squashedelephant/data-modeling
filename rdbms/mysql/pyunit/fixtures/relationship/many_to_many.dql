SELECT e.first_name, e.last_name, d.name FROM employee AS e INNER JOIN dept AS d INNER JOIN employee_dept AS m ON e.id = m.emp_id AND d.id = m.dept_id;
