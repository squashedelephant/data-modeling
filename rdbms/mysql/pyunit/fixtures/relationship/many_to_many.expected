Bill
Gates
Sales
Steve
Jobs
Marketing
Larry
Ellison
Engineering
